mangaki/mangaki/factories.py

```diff
@@ -21,7 +21,6 @@ class Meta:
 
     username = factory.Faker('user_name')
     email = factory.LazyAttribute(lambda o: '{}@mangaki.fr'.format(o.username))
-    profile = factory.RelatedFactory(ProfileFactory, 'user')
```

Comment:
Agreed, can we customize the data inside of the new profile? (in order to generate profiles in different "configuration")?